components/script/window_named_properties.rs

```diff
@@ -113,6 +114,15 @@ unsafe extern "C" fn get_own_property_descriptor(
     return true;
 }
 
+#[allow(unsafe_code)]
+unsafe extern "C" fn own_property_keys(
+    _cx: *mut JSContext,
+    _proxy: HandleObject,
+    _props: MutableHandleIdVector,
+) -> bool {
+    false
```

Comment:
@jdm, not sure if that is the kind of dummy you had in mind.

Reply:
Generally a false return value will be treated as an unrecoverable error by the JS engine,  so you probably want true instead.